gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,7 @@
+#Sat Apr 07 10:05:31 GMT-05:00 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.5.1-bin.zip
-android.enableD8.desugaring=true
\ No newline at end of file
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.5.1-all.zip
```

Comment:
Unless I think this change should be reverted because we don't need the sources and documentation that `all` comes with. Why download things we don't need 😂

Reply:
haha no prob. Never knew that's what the `all` meant. Thanks for pointing that out.